app/jobs/bulkrax/importer_job.rb

```diff
@@ -4,7 +4,7 @@ module Bulkrax
   class ImporterJob < ApplicationJob
     queue_as :import
 
-    def perform(importer_id, only_updates_since_last_import = false)
+    def perform(importer_id, only_updates_since_last_import = false, validate_only = false)
```

Comment:
actually, don't do this, we'll use the validate_only field on the importer itself